docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
-# Confidence Intervals
-By navigating to the detail view of any experiment, you can view the effect that any of the treatments have had on each of the selected metrics for that experiment. You can get to the detail view of any experiment by clicking the **Experiments** tab on the left panel and then clicking on the experiment you're interested in from the list. The effect on each metric comes with an accompanying confidence interval. This confidence interval tells you the upper and lower bound of the effect on the metric in addition to the point estimate of the effect. We show confidence intervals for the ratio between treatment and control, rather than their absolute difference.
+# Lift Estimates and Confidence Intervals
+By navigating to the detail view of any experiment, you can view the effect that any of the treatments have had on each of the selected metrics for that experiment. You can get to the detail view of any experiment by clicking the **Experiments** tab on the left panel and then clicking on the name of the experiment you're interested in from the list. In order to provide effect estimates consistently across all metric types (counts, rates, percentages, etc.), we use the **relative lift** between treatment and control (that is, $\frac{Treatment - Control}{Control}$), rather than their absolute difference.
 
-If the change is positive and a positive change has been marked as good, the interval shows up in green (red if positive is bad). The opposite is true if the change is negative. The interval will be gray if the confidence range includes 0, possibly indicating that the effect of the variation on that metric was not significant.
+The effect on each metric comes with an accompanying **confidence interval**, showing the range of values we consider plausible at the specified **significance level**, which defaults to 5%. The significance level represents how likely a treatment that was identical to the control (that is, an A/A test) would show an effect at least as large as what you've observed in your experiment.[^1] A *higher* significance level means that you are willing to accept more risk of seeing an effect that isn't actually there[^2], but also that you can make a decision with less data; it will also mean that the confidence intervals will be narrower.
```

Comment:
I think we should avoid this language around significance level; the whole point of using confidence intervals is to hide this statistical jargon from folks. 

Instead, we should just rely directly on the definition of a confidence interval: we generate an interval that contains the true underlying value 95% of the time, with slight differences depending on the method chosen (this is not really true for the Bayesian method)

Reply:
I think it's a useful exercise to rewrite this and avoid the term "significance level" and instaed use "confidence level" (95% instead of 5%)

Reply:
Except we use "Significance level" in the product:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/1545637/193650401-e67c3695-5b1d-4b61-b0c4-e16ea76461a4.png">
<img width="257" alt="image" src="https://user-images.githubusercontent.com/1545637/193650449-d4597445-48ec-44df-876f-ebadc853ed09.png">

I agree with this point, but as long as we're using significance level in the product, I think it's more helpful to use the language in the actual product itself.

Reply:
Yup unfortunately we might have to use significance level until we reword that in the app, but we don't have to talk about hypothesis testing is my point, and using confidence level instead of significance level merely a trick to help you reword